### PR TITLE
Updates for accessible color palette

### PR DESCRIPTION
**Pull request by scharlottej13**
- updated the figures used in the main docs with the accessible color palette (see https://github.com/dask/community/issues/135 & https://github.com/dask/dask/pull/8401)
- removed now out of date figures
- included updated brand guidelines, as adapted from this slide deck
[Dask-Brand Guidelines-R2.pdf](https://github.com/dask/marketing/files/7714232/Dask-Brand.Guidelines-R2.pdf)

### CODE REVIEW

**Review by jrbourbeau — COMMENTED** [review]
Thanks @scharlottej13! I just enabled readthedocs builds on pull requests in this repository. Would you mind pushing an empty commit to see is that triggers a build?

```
git commit --allow-empty -m "Trigger CI"
git push origin newer-figures
```

_should_ do the trick

**Comment by scharlottej13** [comment]
> Thanks @scharlottej13! I just enabled readthedocs builds on pull requests in this repository. Would you mind pushing an empty commit to see is that triggers a build?
> 
> ```
> git commit --allow-empty -m "Trigger CI"
> git push origin newer-figures
> ```
> 
> _should_ do the trick

I think that worked! Thanks @jrbourbeau

File: source/colors.rst [hunk 1]
```diff
@@ -1,62 +1,64 @@
...
-
-.. image:: images/dask-colors.svg
-   :alt: Dask colors #FDA061 #ECB172 #D67548
-
-For more involved diagrams we often choose from both primary colors and secondary
-colors:
-
-.. image:: images/dask-branding-guidelines.png
-   :alt: Dask branding guidelines
+These are guidelines for using the Dask logo with community-approved color palettes
```
Comment (scharlottej13) [inline source/colors.rst:4]: @jsignell wanted to flag that I edited this page quite a bit, all info/screenshots are adapted from these slides of updated brand guidelines
[Dask-Brand Guidelines-R2.pdf](https://github.com/dask/marketing/files/7757258/Dask-Brand.Guidelines-R2.pdf)

**Review by jsignell — APPROVED** [review]
This looks good to me!

**Comment by jsignell** [comment]
Thanks for doing this work @scharlottej13! This looks great!

**Pull request merged**